.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Production
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
This should be main, correct?

```suggestion
      - main
```

Reply:
Oh, whoops! Fixed that. I copied the deploy script from constants and forgot to change that.